tests/extendrtests/src/rust/Cargo.toml

```diff
@@ -14,19 +14,24 @@ crate-type = ["staticlib"]
 
 [dependencies]
 extendr-api = "*"
+# Remove "path" when we publish "extendr-graphics" to crates.io
+extendr-graphics = { path = "../../../../../../../extendr/extendr-graphics" }
 
 [patch.crates-io]
 ## This is configured to work with RStudio features.
 ## Replace by absolute path to simplify testing.
 ## CI overrides this path.
-extendr-api = { path = "../../../../../extendr/extendr-api" }
+extendr-api = { path = "../../../../../../../extendr/extendr-api" }
```

Comment:
Honestly, I have no idea why the current path has worked on the CI. I've never succeeded with this on my local...